config/dev.exs

```diff
@@ -44,7 +44,16 @@ config :ex_aws,
 
 config :ueberauth, Ueberauth,
   providers: [
-    cognito: {Skate.Ueberauth.Strategy.Fake, [groups: ["skate-dispatcher", "skate-admin"]]}
+    keycloak: {Skate.Ueberauth.Strategy.Fake, [groups: ["skate-dispatcher", "skate-admin"]]}
+  ]
+
+config :ueberauth_oidcc,
```

Comment:
Are we using Keycloak locally?

Reply:
We're using the fake strategy that always logs the user in, but the key part is that it's called "keycloak" for the purposes of URL paths and such so that no other application code needs to change.